File: app/src/main/java/ufpr/org/churrascometro/calc/BarbecueCalc.java
package ufpr.org.churrascometro.calc;

/**
 * Created by roberto on 18/11/17.
 */

public class BarbecueCalc {

    private double menQuantity;
    private double womenQuantity;
    private double childrenQuantity;

    public BarbecueCalc(double menQuantity, double womenQuantity, double childrenQuantity) {
        this.menQuantity = menQuantity;
        this.womenQuantity = womenQuantity;
        this.childrenQuantity = childrenQuantity;
    }

    public double getMeatQuantity() {
        return ((this.menQuantity * 500) + (this.womenQuantity * 300) + (this.childrenQuantity * 200)) / 1000;
    }

    public double getSausageQuantity() {
        return ((menQuantity * 450) + (womenQuantity * 250) + (childrenQuantity * 200)) / 1000;
    }
}

File: app/src/main/java/com/crave/bbq/calc/BarbecueCalc.kt
package com.crave.bbq.calc

/**
 * Created by roberto on 18/11/17.
 */

class BarbecueCalc(
        private val menQuantity: Double,
        private val womenQuantity: Double,
        private val childrenQuantity: Double
) {

    val meatQuantity: Double
        get() = (this.menQuantity * 500 + this.womenQuantity * 300 + this.childrenQuantity * 200) / 1000

    val sausageQuantity: Double
        get() = (menQuantity * 450 + womenQuantity * 250 + childrenQuantity * 200) / 1000
}
